### PR TITLE
Added support for posting a body of content

### PR DESCRIPTION
**Pull request by drewhemm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,18 @@ modules:
...
+  http_post_2xx:
+    prober: http
+    timeout: 5s
+    http:
+      valid_status_codes: []  # Defaults to 2xx
+      method: POST
+      headers:
+        Content-Type: application/json
+      body: '{}' # Content. Type must match the Content-Type header above
+      no_follow_redirects: false
```
Comment (brian-brazil) [inline README.md]: This can be a more minimal example, the above example covers all these options

File: README.md [hunk 2]
```diff
@@ -131,6 +143,14 @@ scrape_configs:
         regex: .*
         target_label: __address__
         replacement: 127.0.0.1:9115  # Blackbox exporter.
+
+  - job_name: 'blackbox-post'
```
Comment (brian-brazil) [inline README.md]: This will send to prometheus.io, not the blackbox exporter. There's no need for an example here as it's the save as the above.

**Comment by juliusv** [comment]
Ping - someone just asked for this.

**Comment by yosefy** [comment]
yes it is very useful for us

File: README.md [hunk 3]
```diff
@@ -43,6 +43,14 @@ modules:
...
       fail_if_not_matches_regexp:
       - "Download the latest version here"
+  http_post_2xx:
+    prober: http
+    timeout: 5s
+    http:
+      method: POST
+      headers:
+        Content-Type: application/json
+      body: '{}' # Content. Type must match the Content-Type header above
```
Comment (brian-brazil) [inline README.md]: That depends on what you're doing, I'd remove the comment.

**Comment by yosefy** [comment]
very needed for checking api requests

**Comment by yosefy** [comment]
also need some way to debug when not working as expected
cause i have no idea what i receive on post request

File: README.md (hunk 3 above)
Comment (yosefy) [inline README.md]: body: '{}' # Content. Type must match the Content-Type header above

body with Upper B

**Comment by yosefy** [comment]
also i cant understand how to configure prometheus for that
from browser all is working good

**Comment by yosefy** [comment]
Brian, do you plan to merge it
can you share pls

File: README.md (hunk 3 above)
Comment (yosefy) [inline README.md]: will it help to merge if i'll add same PR without mentioned comment and without conflict, as author doesn't seem to answer any type of message?
thanks

Comment (brian-brazil) [inline README.md]: That's be great

Comment (drewhemm) [inline README.md]: Hey guys, sorry, just saw your messages, what do you need me to do?

Comment (yosefy) [inline README.md]: oh hello :)
can you pls see 3 of Brian comments and adjust the commit?
thanks

**Comment by drewhemm** [comment]
I've opened a new PR for this - https://github.com/prometheus/blackbox_exporter/pull/86
Will close this one

File: README.md (hunk 3 above)
Comment (drewhemm) [inline README.md]: Done - sorry for the delay :)

**Pull request closed without merging**